exonum/src/runtime/mod.rs

```diff
@@ -830,6 +848,51 @@ impl<'a> SupervisorExtensions<'a> {
         Dispatcher::initiate_stopping_service(self.0.fork, instance_id)
     }
 
+    /// Initiates resuming previously stopped service instance in the blockchain.
+    ///
+    /// Service will use the specified artifact to resume. Artifact name should be same as
+    /// the artifact name of the stopped instance previously stopped instance.
+    /// Artifact version should be same as the `data_version` stored in the stopped service
+    /// instance.
+    ///
+    /// This method can be used to resume modified service after successful migration.
+    ///
+    /// The service is not immediately activated; it activates if / when the block containing
```

Comment:
"if /" looks excess, IMHO just "when" is enough.

```suggestion
    /// The service is not immediately activated; it activates when the block containing
```